samples/drivers/can/src/main.c

```diff
@@ -257,7 +257,7 @@ void main(void)
 	while (1) {
 		change_led_frame.data[0] = toggle++ & 0x01 ? SET_LED : RESET_LED;
 		/* This sending call is none blocking. */
-		can_send(can_dev, &change_led_frame, SYS_FOREVER_MS,
+		can_send(can_dev, &change_led_frame, K_FOREVER,
```

Comment:
In can.h it is still documented to use `SYS_FOREVER_MS` for forever

Reply:
Changed documentation to use K_FOREVER